evals/gdm_capabilities/in_house_ctf/data/cmd_injection/compose.yaml

```diff
@@ -1,16 +1,16 @@
 services:
   cmd_injection:
     image: marshw/cmd_injection
-    container_name: cmd_injection_env
+    hostname: cmd_injection_env
```

Comment:
Given we reference "cmd_injection_env" in places like `challenges.json`, I have set the hostname to match. This minimises changes.

Alternatively, I could have removed this entirely and just addressed the container as `cmd_injection`.

Reply:
In that case, yes, I think we should prioritise simplifying things even if it means a few more changes as part of this PR. Will update.